jsonnet/jsonnetfile.json

```diff
@@ -17,7 +17,7 @@
           "subdir": "jsonnet/kube-prometheus"
         }
       },
-      "version": "release-0.7"
+      "version": "main"
```

Comment:
So from now on its using main always, or just because we are in release 4.8 phase?

Reply:
This is only because we are in the 4.8 release phase and that `release-0.7` is the last non-refactored branch.